tox.ini

```diff
@@ -1,17 +1,21 @@
 [tox]
 args_are_paths = false
-envlist =
-    py27-{1.7,1.8}
+skip_missing_interpreters = true
+envlist = py{27,33,34,py}-dj{17,18},py35-dj18,flake8
```

Comment:
Turns out this is more than just adding 1.9 to the tox file. I'll fix this in a subsequent PR.